internal/qtest/test_helper.go

```diff
@@ -192,6 +193,13 @@ func WithUserContext(ctx context.Context, user *qf.User) context.Context {
 	return metadata.NewIncomingContext(ctx, meta)
 }
 
+// WithAuthCookie returns context containing an authentication cookie with JWT.
+func WithAuthCookie(ctx context.Context, token string) context.Context {
```

Comment:
Looks like all uses of this helper uses the `interceptor.AuthTokenString()`. Hence, this func should instead provide the following signature:
```go
func WithAuthCookie(ctx context.Context, cookie *http.Cookie) context.Context {
```

See also my other related comment elsewhere regarding moving `interceptor.AuthTokenString()` to `auth` package.